.github/workflows/build_on_request.yml

```diff
@@ -84,7 +83,7 @@ jobs:
   build:
```

Comment:
since there is [a `permission` directive](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions), we can limit the `GITHUB_TOKEN` for this job to `read` (or possibly even `none` since it's a public repo).

Reply:
Will change the token to [restrictive](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token), thanks!

Reply:
All fixed, will port this to all repositories I manage. Thanks again!